tests/test_persistence.py

```diff
@@ -659,12 +659,13 @@ def test_migrate_new(mocker, default_conf, fee, caplog):
                 order_date DATETIME,
                 order_filled_date DATETIME,
                 order_update_date DATETIME,
+                leverage FLOAT,
                 PRIMARY KEY (id),
                 CONSTRAINT _order_pair_order_id UNIQUE (ft_pair, order_id),
                 FOREIGN KEY(ft_trade_id) REFERENCES trades (id)
             )
             """))
-
+        # TODO-mg @xmatthias: Had to add field leverage to this table, check that this is correct
```

Comment:
I would think it's not (not for this test, anyway) - this test tests (try saying that quickly multiple times ..)  database migrations - as such - you'll want to leave the above query untouched - and define the migration correctly in `migrations.py` - which will cause this field to be filled with the correct default value (i assume for leverage it'll be 0  or 1 - but you'll probably also want a "isShort" field in the orders table).

Reply:
So I thought you didn't want leverage and is_short to be in the order class.

The reason I added leverage here is because this test fails without it

Reply:
Yes i know (the failing test is showing that migrations are not running - so by resetting this test to how it was - (maybe even adding an assert to ensure that "leverage" is the default for non-leveraged trades (0, 1 or None?) 

you can test if migrations are running correctly or not (hint: they are currently not - see other comment).